act-rules-format.bs

```diff
@@ -170,12 +205,12 @@ An objective description is one that can be resolved without uncertainty in a gi
 </div>
 
 
-Applicability for Composite Rules {#applicability-composed}
+Applicability for Composite Rules {#applicability-composite}
 ----------------------------------------------------------
 
-A [=composite rule=] defines how the outcomes from its atomic rules are used to determine a single outcome for each applicable test target. 
+A [=composite rule=] defines how the [=outcomes=] from rules in its [atomic rules list](#atomic-rules-list) are used to determine a single outcome. 
 
-The applicability of a composite rule is defined as the union of all the applicability sections of its [=atomic rules=]. Because of this, applicability of a composite rule can be inferred from the atomic rules it uses results from. Since the applicability can be inferred, rule authors MAY omit applicability for a composite rule. This can be useful if it is difficult to express the combined applicability in plain language.
+The applicability of a composite rule MUST be the union of all the applicability sections of the rules in its [atomic rules list](#atomic-rules-list). Because of this, applicability of a composite rule can be inferred from the atomic rules it uses outcomes from. Since the applicability can be inferred, rule authors MAY omit applicability for a composite rule. This can be useful if it is difficult to express the combined applicability in plain language.
```

Comment:
I preferred the previous version, saying “The applicability _is defined as_” rather than “The applicability _MUST be_”: if we say it's _defined_ as such, then there is no room for interpretation, and implementations don't have to guess what to do if the author didn't follow the spec and defined the applicability otherwise. If we use a MUST, then authors may (will) break this statement and implementations are left with an unspecified situation.

Reply:
Decision from the call is to leave the "MUST" in here, it is a requirement for the rule author. We can work on rewording this to address the issue separately.

Reply:
Suggestion: "The applicability of a composite rule is defined as the union of all the applicability sections of its [=atomic rules=]. This can be inferred from the list of atomic rules. However, rule authors MAY provide a description of the applicability for composite rules. This can be useful if it is difficult to express the combined applicability in plain language. In that case, the applicability of the composite rule provided by the author MUST also be the union of all the applicability sections of its."